test/global.test.js

```diff
@@ -1,4 +1,4 @@
-const { detectYarn } = require("../lib/narn-lib");
+const { detectNpm, detectPnpm } = require("../lib/narn-lib");
```

Comment:
Since I changed the **detectYarn**, I had to refactor all the tests, but they do the same basically